src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/AnnotatedVariantProducerUnitTest.java

```diff
@@ -13,233 +15,163 @@
-        data.add(new Object[]{forSimpleTanDupExpansionWithNovelIns_dup_minus,
-                Stream.concat( commonAttributes.stream(), Sets.newHashSet(DUP_TAN_EXPANSION_STRING, DUP_REPEAT_UNIT_REF_SPAN, DUP_SEQ_CIGARS, DUPLICATION_NUMBERS, DUP_ORIENTATIONS, INSERTED_SEQUENCE, INSERTED_SEQUENCE_LENGTH).stream()).sorted().collect(Collectors.toList()),
-                broadcastCNVCalls, referenceBroadcast, refSeqDictBroadcast});
-
-        // tandem dup expansion from 1 unit to 2 units with pseudo-homology
-        data.add(new Object[]{forComplexTanDup_1to2_pseudoHom_plus,
-                Stream.concat( commonAttributes.stream(), Sets.newHashSet(DUP_TAN_EXPANSION_STRING, DUP_REPEAT_UNIT_REF_SPAN, DUPLICATION_NUMBERS, DUP_ANNOTATIONS_IMPRECISE, DUP_ORIENTATIONS, HOMOLOGY, HOMOLOGY_LENGTH, DUP_IMPRECISE_AFFECTED_RANGE).stream()).sorted().collect(Collectors.toList()),
-                broadcastCNVCalls, referenceBroadcast, refSeqDictBroadcast});
-
-        // tandem dup contraction from 2 units to 1 unit with pseudo-homology
-        data.add(new Object[]{forComplexTanDup_2to1_pseudoHom_minus,
-                Stream.concat( commonAttributes.stream(), Sets.newHashSet(DUP_TAN_CONTRACTION_STRING, DUP_REPEAT_UNIT_REF_SPAN, DUPLICATION_NUMBERS, DUP_ANNOTATIONS_IMPRECISE, DUP_ORIENTATIONS, HOMOLOGY, HOMOLOGY_LENGTH, DUP_IMPRECISE_AFFECTED_RANGE).stream()).sorted().collect(Collectors.toList()),
-                broadcastCNVCalls, referenceBroadcast, refSeqDictBroadcast});
-
-        // tandem dup contraction from 3 units to 2 units
-        data.add(new Object[]{forComplexTanDup_3to2_noPseudoHom_plus,
-                Stream.concat( commonAttributes.stream(), Sets.newHashSet(DUP_TAN_CONTRACTION_STRING, DUP_REPEAT_UNIT_REF_SPAN, DUPLICATION_NUMBERS, DUP_ANNOTATIONS_IMPRECISE, DUP_ORIENTATIONS, HOMOLOGY, HOMOLOGY_LENGTH, DUP_IMPRECISE_AFFECTED_RANGE).stream()).sorted().collect(Collectors.toList()),
-                broadcastCNVCalls, referenceBroadcast, refSeqDictBroadcast});
-
-        // tandem dup expansion from 2 units to 3 units
-        data.add(new Object[]{forComplexTanDup_2to3_noPseudoHom_minus,
-                Stream.concat( commonAttributes.stream(), Sets.newHashSet(DUP_TAN_EXPANSION_STRING, DUP_REPEAT_UNIT_REF_SPAN, DUPLICATION_NUMBERS, DUP_ANNOTATIONS_IMPRECISE, DUP_ORIENTATIONS, HOMOLOGY, HOMOLOGY_LENGTH, DUP_IMPRECISE_AFFECTED_RANGE).stream()).sorted().collect(Collectors.toList()),
-                broadcastCNVCalls, referenceBroadcast, refSeqDictBroadcast});
-
-        return data.toArray(new Object[data.size()][]);
+        Broadcast<ReferenceMultiSource> referenceBroadcast = testSparkContext.broadcast(TestUtilsForAssemblyBasedSVDiscovery.b38_reference_chr20_chr21);
+        Broadcast<SAMSequenceDictionary> refSeqDictBroadcast = testSparkContext.broadcast(TestUtilsForAssemblyBasedSVDiscovery.b38_seqDict_chr20_chr21);
+
+        // multiple evidence contigs
```

Comment:
I'm not sure what these comments are referring to, can you remove or expand on them?